packages/beacon-node/src/sync/utils/downloadByRange.ts

```diff
@@ -375,20 +381,30 @@ export function validateBlockByRangeResponse(
   config: ChainForkConfig,
   blocksRequest: phase0.BeaconBlocksByRangeRequest,
   blocks: SignedBeaconBlock[]
-): ValidatedBlock[] {
+): WarnResult<ValidatedBlock[], DownloadByRangeError> {
```

Comment:
I don't see why `WarnResult` is used here, looks like it's only used in the commented code